cmd/govim/oswatcher.go

```diff
@@ -0,0 +1,47 @@
+// +build !darwin
+
+package main
```

Comment:
I'd be half tempted to move this to an `internal` package now that things are growing a bit. 

That will also help to isolate what we're at some point going to throw away, too, keeping it separate from the code we will keep

Reply:
The entire thing? I.e. both `watcher.go` and `oswatcher*.go` (terrible name, feel free to suggest a better if you come up with one).